### PR TITLE
Feature/233/about-page-alignment

### PR DESCRIPTION
**Pull request by aliceluo2048**
### Describe the problem being solved:

- Changed photos dimensions to be 400px by 400px 
- Aligned the top line of the respective team member's info content to the top of their photo
- Increased spacing between each team member
- Changed the heading size in mobile/tablet css to be consistent with alignment

Before:
before-desktop

After: (orange highlight to demonstrate the alignment) 

after-desktop

**Mobile/tablet changes:**
Before:
before-responsive

After:
responsive-after

### Impacted areas in the application: 

List general components of the application that this PR will affect: 
* About us page

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

### CODE REVIEW

File: frontend/src/components/static_pages/about.css [hunk 1]
```diff
@@ -6,17 +6,23 @@
...
   width: 400px;
+  height: 400px;
 }
 
 .sub-heading {
   font-size: 24px;
 }
 
+.people-heading {
+  font-size: 36pt;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: lets use `48px` here. Its the same as `36pt` in terms of size but `px` is the best practices as `pt` is more of a print measurement.

File: frontend/src/components/static_pages/about.css [hunk 2]
```diff
@@ -35,6 +41,10 @@
   .heading {
     font-size: 30px;
   }
+  .people-heading {
+    font-size: 30px;
+    font-weight: bold;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: this the same as the style without the screen check, so it is unneeded. 

Only need to add styles in here that are different like how the font size is changing

File: frontend/src/components/static_pages/about.css [hunk 3]
```diff
@@ -6,17 +6,23 @@
 
 .images {
   margin: 0;
-  margin-bottom: 40px;
+  margin-bottom: 140px;
 }
 
 .images img {
   width: 400px;
+  height: 400px;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: this doesn't look great at a large desktop size. I think we just want this for screen sizes at 1023px and below? Maybe we should talk with Tina about this requirement

File: frontend/src/components/static_pages/about.css (hunk 2 above)
Comment (aliceluo2048) [reply frontend/src/components/static_pages/about.css]: On hold until further details on how to treat the photos.

File: frontend/src/components/static_pages/about.css (hunk 3 above)
Comment (aliceluo2048) [reply frontend/src/components/static_pages/about.css]: On hold until further details on how to treat the photos.

Comment (aliceluo2048) [reply frontend/src/components/static_pages/about.css]: Photos have been edited to be 400px x 400px. Please review changes.

**Pull request merged**